src/publishProject/formComponentHelpers.ts

```diff
@@ -3,36 +3,89 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { FormItemType } from "../sharedInterfaces/form";
+import * as vscode from "vscode";
+import * as constants from "../constants/constants";
+import { FormItemType, FormItemOptions } from "../sharedInterfaces/form";
+import { PublishProject as Loc } from "../constants/locConstants";
 import {
     IPublishForm,
     PublishDialogFormItemSpec,
     PublishDialogState,
+    PublishTarget,
 } from "../sharedInterfaces/publishDialog";
-import { PublishProject as Loc } from "../constants/locConstants";
+import { getPublishServerName, validateSqlServerPortNumber } from "./projectUtils";
+import { validateSqlServerPassword } from "../deployment/dockerUtils";
+
+/**
+ * Checks if preview features are enabled in VS Code settings for SQL Database Projects.
+ * @returns true if preview features are enabled, false otherwise
+ */
+function isPreviewFeaturesEnabled(): boolean {
+    return (
+        vscode.workspace
+            .getConfiguration(constants.DBProjectConfigurationKey)
+            .get<boolean>(constants.enablePreviewFeaturesKey) ?? false
```

Comment:
Looks like this feature flag is just for the SqlProj features.  Can you rename the variable name to something like `enableSqlProjPreviewFeaturesKey`?